### PR TITLE
add basic UI

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
- Created `app.rb` file that will serve as your console app entry-point. 
- In `main.rb` defined the entry point, this will be a method called `main` that is invoked at the end of the file. This method contains:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,189 @@
+$LOAD_PATH << '.'
+require 'classroom'
+require 'person'
+require 'teacher'
+require 'student'
+require 'book'
+require 'rental'
+require 'utils'
```
Comment (Kingstalux) [inline app.rb:8]: - [x] Good job so far 👍 You do not have any files named `utils` in your directory, Kindly correct this name to `main` as I believe that is the file you are trying to access.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+module Main
```
Comment (Kingstalux) [inline main.rb]: - [x] Kindly change this module name to `Utils` as that is the name you've been using in your `app.rb` file.

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    has_parent_permission = Utils.read_permission == 'Y'
+    [age, name, has_parent_permission]
+  end
+
+  def teacher_info
+    age = Utils.read_age
+    name = Utils.read_name
+    specialization = Utils.read_specialization
+    [age, name, specialization]
+  end
```
Comment (Kingstalux) [inline app.rb:39]: - [ ] All methods work except the method of creating a teacher. I'm having the following error when I try to create a teacher. Kindly rectify that 👍
- Error:
Annotation 2022-05-13 082714

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    end
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
+
+main
```
Comment (Kingstalux) [inline app.rb:189]: - [ ] This code should be transferred to the `main.rb` file so that your console app can be started using the `ruby main.rb`

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Dagic-zewdu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966

## Highlights 👏 
✅ Good pull request description 👍 
✅ Linters are passing 🔥 
✅ Professional README file 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Kingstalux) [reply app.rb:8]: resolved 👍

File: main.rb (hunk 2 above)
Comment (Kingstalux) [reply main.rb]: resolved 👍

File: app.rb (hunk 3 above)
Comment (Kingstalux) [reply app.rb:39]: resolved 👍

File: app.rb (hunk 4 above)
Comment (Kingstalux) [reply app.rb:189]: not resolved

Comment (Kingstalux) [inline app.rb:189]: - [ ] Great job so far 👍 Your app functions perfectly and you are almost there. Kindly transfer this code to a `main.rb` file which will serve as the entry point to your console app when a user inputs `ruby main.rb`. You can read more on entry points [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point).

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Dagic-zewdu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f5754646735474252343558364e6278714a4b2f67697068792e676966


 ## Highlights 👏 
✅ App has the correct functionality 🔥 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Dagic-zewdu 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**